### DISCUSSION TITLE
Wrapper for React / compatibility with reactchartjs/react-chartjs-2?

### DISCUSSION THREAD

**Discussion by lucasrdrgs** [kurkle/chartjs-chart-matrix#39] [Q&A] [answered]
I was wondering if this extension works with React, preferably in a way like `react-chartjs-2` (a component for the matrix chart). Can anyone enlighten me, please? Thanks!

**Comment by joshuaharveykeller** [comment, marked as answer]
See this previously closed issue with a similar question:

[Trouble running under React](https://github.com/kurkle/chartjs-chart-matrix/issues/11)
